app/src/main/java/org/simple/clinic/summary/bloodsugar/view/BloodSugarSummaryView.kt

```diff
@@ -58,7 +63,15 @@ class BloodSugarSummaryView(
     LayoutInflater.from(context).inflate(R.layout.patientsummary_bloodsugarsummary_content, this, true)
   }
 
-  var addNewBloodSugarClicked: AddNewBloodSugarClicked? = null
+  private val events: Observable<BloodSugarSummaryViewEvent> by unsafeLazy {
+    addNewBloodSugarClicks()
+        .compose(ReportAnalyticsEvents())
+        .cast<BloodSugarSummaryViewEvent>()
+  }
+
+  private fun addNewBloodSugarClicks(): Observable<BloodSugarSummaryViewEvent> {
+    return RxView.clicks(newBloodSugar).map { NewBloodSugarClicked }
```

Comment:
Please use RxBindings3 for this `newBloodSugar.clicks().map { NewBloodSugarClicked }`